.travis.yml

```diff
@@ -220,6 +221,14 @@ install:
       cmake --version;
     fi
 
+  - if [[ "$crypto" == "wolfssl" ]]; then
+      wget --quiet -O wolfssl-4.2.0-stable.zip https://github.com/wolfSSL/wolfssl/archive/v4.2.0-stable.zip;
+      unzip wolfssl-4.2.0-stable.zip;
+      cd wolfssl-4.2.0-stable;
+      ./autogen.sh && ./configure --enable-asio && make && sudo make install;
```

Comment:
what does `--enable-asio` do? If the installed version of wolfSSL needs to be built with a specific configuration, it's probably a good idea to mention this in the documentation somewhere

Reply:
I was surprised by the need of this option, per wolfSSL documentation, it's needed to put all the OpenSSL layer in place in order to ASIO be able to work with it, but in fact it just enable "opensslextra" and "opensslall", and that makes `libwolfssl` quite big, but still significantly smaller than openssl